mulighetsrommet-api/src/main/resources/web/openapi.yaml

```diff
@@ -1275,6 +1278,16 @@ components:
       required:
         - navn
         - enhetId
+    GeografiskEnhet:
```

Comment:
Er typene `Oppfolgingsenhet` og `GografiskEnhet` faktisk forskjellige (navn + enhetId vs navn + enhetsnummer), eller kunne disse med fordel delt samme modell, f.eks. `NavEnhet`?

Reply:
De er forskjellige. Fra veilarboppfolging som vi får oppfolgingsenhet fra så er det navn + enhetId, mens for veilarbperson og geografisk enhet er det navn + enhetsnummer